corehq/motech/generic_inbound/views.py

```diff
@@ -242,13 +242,14 @@ def _log_api_request(api, request, response):
         is_success = False
         status = RequestLog.Status.ERROR
 
-    response_body = response.content.decode('utf-8')
+    # TODO, revisit this
```

Comment:
Looking for input here (perhaps best to revisit this comment after reviewing the rest of the PR).  Given that the "response" is JSON, does it make sense to store the raw response body?  At the moment it's a deterministic transform of the response JSON.  I understand in the future we might want to include a response transform function, in which case storing before and after that does make sense, but it feels vague enough that I'm not sure it make sense to sorta implement it now.

Reply:
> I understand in the future we might want to include a response transform function, in which case storing before and after that does make sense, but it feels vague enough that I'm not sure it make sense to sorta implement it now.

I agree that it doesn't make a lot of sense right now. Fine to drop it or at least remove it's display from the UI since it is a bit confusing. I was also thinking of renaming it in the UI to clarify the difference:
`raw response` -> `Case API response`
`response body` -> `Response to Client`